apps/example/webapps/showcase/previews/ack-button.pug

```diff
@@ -1,5 +1,5 @@
-panel(type="info" title="ack-button")
-    .content
+.col-md-12
```

Comment:
We should move this kind of definition into `page` component so that we don't have to replace in all applications.